### PR TITLE
Add bionicle4365's Bear Grass file

### LINKED ISSUE #7: bionicle4365's Workflow for adding Bear Grass to the Repository
## Bear Grass, NC
- [x] Create an Issue
- [x] Create a branch
- [x] Make changes
- [x] Open a PR
- [x] Collaborate/Have a discussion
- [x] Get approval to merge
- [x] Merge

### PR DESCRIPTION
**Pull request by bionicle4365**
closes #7

### CODE REVIEW

File: bionicle4365-BearGrass.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Bear Grass, NC
+
```
Comment (smurffy) [inline bionicle4365-BearGrass.md:2]: I have never heard of Bear Grass before 🐻

File: bionicle4365-BearGrass.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+### Restaurants
+
+- None
+
+### Places To Go
+
+- None
```
Comment (smurffy) [inline bionicle4365-BearGrass.md:9]: sounds like there is nothing to do there, bummer 🙇

**Comment by smurffy** [comment]
Are there any bears in Bear Grass, NC?

**Comment by bionicle4365** [comment]
Many! They would climb trees in my neighborhood :fearful:

**Pull request merged**